user_guide/feature_contexts/hooking_into_the_test_process.rst

```diff
@@ -226,6 +234,34 @@ same as a usual scenario.
 ``@AfterScenario`` functions exactly the same way, being executed both after
 usual scenarios and outline examples.
 
+Feature Hooks
```

Comment:
Can you be a bit more specific on what I have to modify? I agree that that order does not make sense. I think it should be `Suite > Feature > Scenario > Step` .. But I'm not 100% sure that is what you're saying here.